.changeset/blue-falcons-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"app-reglementaire-bijlage": minor
```

Comment:
I don't have a good conception of how release versions should work in the app repos, but shouldn't this be a major release as it requires an update in app-gn to have a version of the editor which is compatible with the snippet changes?

Reply:
I'm not really sure if we really have a versioning strategy for the apps/frontends, as it is less important than our npm packages. I'm not sure whether it matters a lot, but I'm always in favour of determining a good version strategy for the apps too.